src/config/mod.rs

```diff
@@ -88,21 +88,21 @@ impl<'c> Config<'c> {
     }
 
```

Comment:
Some rationale would be good to have on " Put relative paths starting with / into BootSource objects" commit.